src/psyclone/gocean1p0.py

```diff
@@ -1109,6 +1109,52 @@ def __init__(self):
         self._name = ""
         self._index_offset = ""
 
+    @staticmethod
+    def _format_access(var_name, var_value, depth):
```

Comment:
I was actually thinking about this as well, but changing from strings to symbols is a much bigger change (and if we would only do it partially we would break a lot of code, of would have to add if-tests to see if we have a string of symbol). I've added a reference to #845 here (and if you think we are far enough with using symbols that we can switch, I'll do #845 next).

Reply:
Thanks Joerg. We'll maybe need to discuss with Sergi at the next telco and take stock of where we're up to. (I think he has the PSy-layer generation working with the PSyIR in GOcean now but can't [to my shame] remember whether this is all done and on master).